### ISSUE TITLE
[ALM Accelerator - QUESTION] What is the best approach to testing new versions of ALM?

### ISSUE THREAD

**Issue by tocri** [microsoft/coe-starter-kit#6896]
### What is your question?

I am looking for the best approach to test new versions of ALM without breaking my project. 
To date, I use 2 deployment profiles :

* 1 for the organization's dev
* 1 for the citizen dev. 
*
* Each of these profiles points to a specific repo.

I can't create another project just for my tests because my devops administrator doesn't want to.

What I have set up for the moment:

* 1 lab environment that is used in which we host our TEST solutions
* 1 ALM TEST environment that is used to test the software updates.

What I am questioning is what is the good practice to test ALM without breaking my project.

Should I create a new repo in my project that I could name "Testing ALM Updates only"?

Should ideally create a new project?

In short, how do you do it?

### What component are you experiencing the issue with?

ALM Accelerator Canvas App

### What solution version are you using?

october 23

AB#261

**Comment by RajeevPentyala** [comment]
@tocri The recommendation is to have a dedicated Dataverse environment and a dedicated AzDo project. If your policies do not permit having a dedicated AzDo project, you might want to go with a [Developer Environment](https://learn.microsoft.com/en-us/power-platform/developer/create-developer-environment) and with the same developer account, you can spin up a new Azure DevOps organization. This way you would be able to isolate and test the latest builds.

We don't recommend using a single Azure DevOps project for both past and current builds. The reason is that the 'Export Pipeline' (i.e., export-solution-to-git) is set at the project level and is linked to a YAML file in one of the repositories. It gets complicated to maintain 2 'Export Pipeline's pointing to 2 repositories one with old build and one with latest build.

Please reach out if you have any questions.

**Comment by RajeevPentyala** [comment]
@tocri I hope you have the necessary details. Closing this question, but feel free to reach out if you need more information.
